shared/desktop/renderer/main2.desktop.tsx

```diff
@@ -80,9 +80,7 @@ const setupApp = (store, runSagas) => {
 
   // See if we're connected, and try starting keybase if not
   setTimeout(() => {
-    if (!eng.hasEverConnected()) {
-      SafeElectron.getApp().emit('KBkeybase', '', {type: 'requestStartService'})
-    }
+    SafeElectron.getApp().emit('KBkeybase', '', {type: 'requestStartService'})
```

Comment:
Can you explain this change? commit says "always request a full service start on windows when the gui spins up". I'm guessing this is to prevent situation when GUI starts, there is only the service running for some reason (hasEverConnected returns true), and we don't start KBFS this way?

``hasEverConnected` method seems unused after this change.

Reply:
your assumption is correct. before this change, if you have a service running but no kbfs (e.g. autoforked by running something like `keybase id` when nothing at all is running), then you start the gui, nothing will have started kbfsdokan or the updater or the watchdog. and you'll blackbar if you go to the files tab. so yes. this change does exactly what you think.

the hasEverConnected. i'll look into that.

Reply:
ok. yeah. i think i can delete a little more code here. 👍

Reply:
done

Reply:
lets only make windows changes in this PR so keep this logic for other platforms

Reply:
@chrisnojima it looked to me like the only code responding to `requestStartService` is something that already limits to windows only: https://github.com/keybase/client/blob/master/shared/desktop/app/node.desktop.tsx#L272. maybe i'm missing something. or if you'd prefer we move that `if windows` up in the flow to here, i can do that too.

Reply:
something should be done to make it more clear. if its disconnected by which flows happen to send this message or handle it its not 1. enforced in any way and 2. not clear this is the case. 
Or we should rename it so its more specific like requestWindowsStartService or something